Branch-SDK/Branch-SDK/BNCStrongMatchHelper.m

```diff
@@ -112,11 +112,23 @@ - (void)presentSafariVCWithBranchKey:(NSString *)branchKey {
         
         // Must be on next run loop to avoid a warning
         dispatch_async(dispatch_get_main_queue(), ^{
-            [windowRootController presentViewController:safController animated:YES completion:^{
-                [self.secondWindow.rootViewController dismissViewControllerAnimated:YES completion:NULL];
-                [BNCPreferenceHelper preferenceHelper].lastStrongMatchDate = [NSDate date];
-                self.requestInProgress = NO;
-            }];
+            // Add the safari view controller using view controller containment
+            [windowRootController addChildViewController:safController];
+            [windowRootController.view addSubview:safController.view];
+            [safController didMoveToParentViewController:windowRootController];
+            
+            // Remove the safari view controller from view controller containment
```

Comment:
Unfortunately we can't do that because of the way we're using SFSafariViewController. If we required everyone to link their project to SFSafariServices / use Xcode 7+, then we could write clearer code instead of using `NSClassFromString`, but we want to keep that optional.